function handleError(error, message) {
    console.error(message)
    console.error(error)
}

module.exports = handleError;